.gitignore

```diff
@@ -9,3 +9,5 @@
 *data/
 *checkpoints/
 *.pth
+
+!/integrations-and-supported-tools/sagemaker/data/
```

Comment:
Instead of pushing the dataset to the repo, lets read directly from https://gist.githubusercontent.com/curran/a08a1080b88344b0c8a7/raw/0e7a9b0a5d22642a06d3d5b9bcbad9890c8ee534/iris.csv during runtime

Reply:
I will download the file from AWS SageMaker examples S3 bucket.